src/client.rs

```diff
@@ -373,6 +375,10 @@ impl<C: Config> Client<C> {
         self.msgs_to_send.push(msg.into());
     }
 
+    pub fn send_plugin_message(&mut self, msg: PluginMessageToClient) {
+        self.plugin_messages_to_send.push(msg);
```

Comment:
Any reason you can't send the plugin message packet here directly? Does the plugin message packet work if sent before the login/join game packet? Also this function should take the channel and data as args to avoid exposing the `PluginMessageToClient` in the public API.

Reply:
> Any reason you can't send the plugin message packet here directly? Does the plugin message packet work if sent before the login/join game packet?
While in login state there are `Login Plugin Request` and `Login Plugin Response` packets, which are limited to server requests and clients responses. Thats why i'm not sure if directly sending it would work.

Reply:
By the time you have a `Client` you're already past the login stage and into the play stage so I think it would work.